data_generators/xcspec_extractor.py

```diff
@@ -63,29 +65,147 @@ def xcspec_from_file(path):
     "com.apple.xcode.tools.swift.compiler",
 )
 
-bzl_contents = """
-# Extracted from Xcode %s
+header = """
+# Extracted from Xcode {xcode_version}
 # To update, in rules_ios run `bazel run data_generators:extract_xcspecs`
+""".format(xcode_version=xcode_version)
```

Comment:
Let's add a much bigger warning here. It's easy to miss that those two files are generated code.

```suggestion
header = """
#########################################################
#                                  THIS IS GENERATED CODE                                             #
# Extracted from Xcode {xcode_version}                                                           #
# To update, in rules_ios run `bazel run data_generators:extract_xcspecs`  #
#########################################################
""".format(xcode_version=xcode_version)
```